---
title: "13-08-19 Quast-lg Assembly Analysis"
layout: single
---

## Purpose: 


### Command   
```
quast --large --eukaryote --k-mer-stats --circos --gene-finding --threads 10 --conserved-genes-finding /media/jeilers/Elements/P_cali_genome/Working Files/P_cali-8.fa
```

### Results  

Below are some of the results from the intial analysis. For a better formated and more details view the report [here](/Research Documents/quast_results/results_2019_08_12_21_35_11/report.html/)

|Number of genes|Percent off total|Description|
|124|41.0%|	Complete BUSCOs (C)|
|89|29.4%|	Complete and single-copy BUSCOs (S)|
|35|11.6%|	Complete and duplicated BUSCOs (D)|
|89|29.4%|	Fragmented BUSCOs (F)|
|90|29.6%|	Missing BUSCOs (M)|
|303||	Total BUSCO groups searched|


### Thoughts
* haploid genome size is predicted to be ~468 mb or diploid at ~936 mb
* 41% of conserved buscos are present, this is actually rather low. Considering this assembly only includes short sequence data though and hasn't been optimized or trimmed/cleaned this is probably not all that bad. 

### Notes  
* I hadn't set up proper license permissions for genemark so it wasn't used.
* Not sure why a circos plot wasn't created. Although, not sure why it would be either
* quast can also use multiple reference genomes/genes databases for analysis. I think the next round I will use the reference genomes/genes from both *Apostichopus japonicus* and *Parastichopus parvimensis*
* Apparently quast also uses redundans for reference assisted analysis. I was planning on using it for reference assisted assembly. Kinda cool. 
* The quast log can be found [here](/Research Documents/quast_results/results_2019_08_12_21_35_11/quast.log/)
