
Notes: I need to rerun fastqc with kmer content and per sequence GC content

### Resources 
An good walk through of the different fastqc outputs and their meaning in addition to quality control processing: [FastQC info by open repository](https://openrepository.aut.ac.nz/bitstream/handle/10292/5170/FASTQC%20analysis%20guide.pdf?sequence=5&isAllowed=y)


Excellent comparisons of bad versus good quality sequence data and the graphs fastqc produces: [Fastqc tutorial and faq by MSU](https://rtsf.natsci.msu.edu/genomics/tech-notes/fastqc-tutorial-and-faq/)


Also contains excellent comparisons of bad versus good quality sequence data along with some processing steps: [Bioinformatics Core Training](https://bioinformatics-core-shared-training.github.io/cruk-autumn-school-2017/Introduction/SS_DB/Materials/Lectures/Lecture2_qualityControl_artefactRemoval_DB.pdf)
