---
toc: true
---

## Why are we interested in PPIs

Over the past twenty years, our understanding of cellular processes has grown more
detailed: from the gene inventory of the Human Genome Project to the advent
of high-throughput transcriptomics. However, in most cases, mRNA remains an
intermediate layer between the genes and proteins, the functional units of the cell.
As such, there is a need to study the proteome at greater depth and breadth to
better understand the intermediary causal molecular pathways between genotypes
and phenotypes. Proteins are involved in countless biological functions and perform
their roles in concert, by interacting together in chains or complexes to perform
biochemical reactions, and it is thus unsurprising that malfunctioning proteins are
implicated in numerous diseases, and therefore, often in their treatment as well.

Unfortunately, though the proteome is perhaps the most interesting level of cellular
systems, it is also the most complex. There are an estimated 650,000 human protein-protein
interactions (PPIs) which are dynamic and occur in a wide range of forms:
some proteins build strong and stable complexes, other weak transient interactions.
This complexity and variability of the proteome makes measurements complicated
compared to advanced sequencing methods and even though it is starting to
change with high-throughput detection methods, the data obtained remains flawed.
The two main methods used are yeast-two-hybrids (Y2H) to uncover physical binary
interactions and co-immunoprecipitation and mass spectrometry (MS)-based
methods to identify protein complexes. The challenge with these methods is to
reconcile unbiased, large-scale testing with in vivo vs in vitro disparities: this leads
to a high incidence of false positive and false negative results. Such results may be
due to self-activating baits, an insufficient depth of screening or the inaccurate
representation of protein complexes. Furthermore, this data is expensive
to collect, both in time and resources; especially considering the magnitude of the
proteome: with about 20,000 human proteins, it is more than 200 million potential
pairwise interactions that need to be tested. Therefore, only about 20% of interactions
are analysed by these high-throughput methods and the selection is biased
towards well-studied or highly expressed proteins. Hence, there is a need for
computational methods to deliver a more accurate and comprehensive view of the
protein-protein interaction network.

Identifying disease-related protein interactions can have a significant impact on drug
development; the Nutlins inhibitors are a good example of what can be achieved
by analysing PPIs: overexpression of MDM2 is a way in which certain cancer cells
avoid the apoptosis-triggering activity of P53 but the Nutlins have been shown capable
of activating cell cycle arrest and apoptosis by binding to MDM2. This example shows
the potential impact of a systematic way to identify these critical interactions which
can then be used as therapeutic targets.
